### ISSUE TITLE
Enhancement: Display progress of cut operation

### ISSUE THREAD

**Issue by mbs** [mifi/lossless-cut#37]
Enhancement request: Display progress of cut operation, either with progress bar, percentage complete, or estimated time to completion.

**Comment by mifi** [comment]
How big are the files you are trying to cut and how long is it taking?

We could do this by using ffmpeg's `-progress` option. Easiest way would be to use /dev/stdout, but that's not possible on windows, so would need to write to temporary files.

-progress outputs such sequences:
```
frame=167524
fps=22331.2
stream_0_0_q=-1.0
bitrate=2179.0kbits/s
total_size=1903342573
out_time_ms=6988052500
out_time=01:56:28.052500
dup_frames=0
drop_frames=0
speed= 932x
progress=continue
frame=175560
fps=22277.1
stream_0_0_q=-1.0
bitrate=2199.4kbits/s
total_size=2013375490
out_time_ms=7323306500
out_time=02:02:03.306500
dup_frames=0
drop_frames=0
speed= 929x
progress=end
```

We could take out_time_ms and divide by the total time of the clip to get the fractional progress.

**Comment by mbs** [comment]
The cases I was looking at were tens of seconds, but I think that could go up significantly with typical filesizes depending on the job configured and output media.

**Comment by mifi** [comment]
Oh. Because all files i tried, even hour long video clips take only a few seconds to cut. Maybe you are cutting raw videos that are hundreds of gigabytes?

**Comment by mbs** [comment]
These were ordinary size, but I may have slower-than-typical storage.  If you cut a 1G original down to a 900M clip, that is going to take a while on typical storage, I think, due mostly to the output IO needed.

**Comment by mifi** [comment]
Yes I'm cutting on a mac with superfast SSD, so i guess thats why it's so fast here.

**Comment by mifi** [comment]
Implemented in d208270cd0d5225ed43c6529b13249575a05a75f. will be included in next release
